src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -565,4 +572,39 @@ export class TeamlistComponent implements OnInit, OnDestroy {
     };
     this.globalService.showTermsAndConditionsModal(PARAMS);
   }
+
+  /**
+   * Display confirm dialog before deleting a team member.
+   */
+  deleteTeamMemberWrapper() {
+    const SELF = this;
+    const deleteTeamMember = (team) => {
+      const deleteUrl = SELF.deleteMembersPath.replace('<team_id>', team.teamId);
+      SELF.apiCall = (params) => {
+        SELF.apiService.deleteUrl(deleteUrl + team.participantId).subscribe(
+        data => {
+          // Success Message in data.message
+          SELF.globalService.showToast('success', 'Member was removed from the team!', 5);
+          SELF.fetchMyTeams(SELF.fetchTeamsPath);
+          SELF.selectedTeam = null;
+        },
+        err => {
+          SELF.globalService.handleApiError(err);
+        },
+        () => {}
+        );
+      };
+      const PARAMS = {
+        title: 'Would you like to remove this member ?',
+        content: 'Note: This action will remove this member from the team.',
```

Comment:
@Kajol-Kumari Don't you think we should also modify this message when host tries to remove himself as it will delete the team directly instead of removing the host only.

Reply:
@Sanji515 If a host tries to remove himself from the team. The request will be aborted and a message is being shown saying that `if you want to delete yourself, instead you delete the team` as shown in the GIF above.

Reply:
Yeah but I've just checked on my local instead of showing the message it directly deletes the team saying `Member removed from the team!`. Can you please check this once @Kajol-Kumari

Reply:
yes @Sanji515 that is because of the bug i have mentioned in [here](https://github.com/Cloud-CV/EvalAI/issues/2664) which i have fixed in [this](https://github.com/Cloud-CV/EvalAI/pull/2781)PR and i have used that branch as backend for working on this PR.

Reply:
I see. But can't we add that check on the frontend itself. As we know who the host is and then we can just check the user if it's matches to the host user.

Reply:
okay lets add this in a separate PR as it may require some other variables for the check.